src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -23,6 +23,7 @@ namespace OpenTelemetry.Metrics
     internal sealed class AggregatorStore
     {
         private static readonly string MetricPointCapHitFixMessage = "Modify instrumentation to reduce the number of unique key/value pair combinations. Or use Views to drop unwanted tags. Or use MeterProviderBuilder.SetMaxMetricPointsPerMetricStream to set higher limit.";
+        private static readonly Comparison<KeyValuePair<string, object>> DimensionComparisonDelegate = DimensionComparison;
```

Comment:
nit: I think you could also declare this like...

```csharp
        private static readonly Comparison<KeyValuePair<string, object>> DimensionComparisonDelegate
            = (x, y) => x.Key.CompareTo(y.Key);
```

Simple enough little method it might read slightly better that way 🤷‍♀️